src/sage/knots/knotinfo.py

```diff
@@ -1738,6 +1748,35 @@ def khovanov_polynomial(self, var1='q', var2='t', base_ring=ZZ, original=False):
             sage: L = KnotInfo.L5a1_0
             sage: Lk = L.khovanov_polynomial(); Lk
             q^4*t^2 + t + 2 + 2*q^-2 + q^-2*t^-1 + q^-4*t^-2 + q^-6*t^-2 + q^-8*t^-3
+            sage: L.khovanov_polynomial(original=True)
+             '2 + 2/q^2 + 1/(q^8*t^3) + 1/(q^6*t^2) + 1/(q^4*t^2) + 1/(q^2*t) + t + q^4*t^2'
+
+        Obtaining the reduced homology (for knots only)::
+
+            sage: Kkr = K.khovanov_polynomial(reduced=True); Kkr
+            q^6*t^3 + 2*q^4*t^2 + 2*q^2*t + 3 + 2*q^-2*t^-1 + 2*q^-4*t^-2 + q^-6*t^-3
+            sage: K.khovanov_polynomial(base_ring=QQ, reduced=True) == Kkr
+            True
+            sage: Kkr2 = K.khovanov_polynomial(base_ring=GF(2), reduced=True); Kkr2
+            q^6*t^3 + 1 + q^-6*t^-3
+            sage: KnotInfo.K8_19.inject()                               # optional database_knotinfo
+            Defining K8_19
+            sage: K8kr = K8_19.khovanov_polynomial(reduced=True); K8kr  # optional database_knotinfo
+            q^16*t^5 + q^12*t^4 + q^12*t^3 + q^10*t^2 + q^6
+
+        Obtaining the odd Khovanov homology (for knots only)::
+
+            sage: K.khovanov_polynomial(odd=True) == Kkr
+            True
+            sage: K.khovanov_polynomial(base_ring=QQ, odd=True) == Kkr
+            True
+            sage: K.khovanov_polynomial(base_ring=GF(2), odd=True) == Kkr2
+            True
+            sage: K8ko = K8_19.khovanov_polynomial(odd=True); K8ko     # optional database_knotinfo
+            q^16*t^5 + q^10*t^2 + q^6
+            sage: K8kr == K8ko                                         # optional database_knotinfo
+            False
+
```

Comment:
One too many blank lines. `;)`

Reply:
Amazingly, pycodestyle doesn't recognize this (probably because it's in the doctest).

Reply:
I've added a fix in [this commit](https://github.com/soehms/sage/commit/3291f4d1662405303ea95e50db81b59a8a563ec0) but is not shown here.